src/app/globals.css

```
@@ -7,6 +7,9 @@
 @custom-variant dark (&:is(.dark *));
 
 @theme {
+  --font-sans: var(--font-noto-sans-jp), "Hiragino Sans", "Hiragino Kaku Gothic ProN",
+    "Yu Gothic", Meiryo, system-ui, sans-serif;
```

Stylelint (17.11.1)

[error] Expected "Meiryo" to be "meiryo" (value-keyword-case)

(value-keyword-case)